cl/recap/api_serializers.py

```diff
@@ -151,6 +153,24 @@ def validate(self, attrs):
 
         return attrs
 
+class EmailProcessingQueueSerializer(serializers.ModelSerializer):
+    # Powers the RECAP upload endpoint
+    mail = serializers.JSONField()
+    receipt = serializers.JSONField()
+
+    def validate(self, attrs):
+        # TODO - Some kind of validation for require properties in mail and receip structure.
+        return attrs
+
+    def create(self, validated_data):
+        mail = validated_data.get('mail')
+        receipt = validated_data.get('receipt')
+
+        # TODO - Parsing logic here. Find the court, upload the file, apply the uploader, create recap documents.
```

Comment:
This is the big question. Should the vast majority of functionality for creating this model live in the Serializer create override?

Reply:
No. We want to do almost no work here. Generally, the only thing we want to do when we get a valid POST is enqueue a task in Celery. Inside that task we'll do all the work. 

There's a fuzzy line somewhere about how much validation to do, but I would definitely draw that line short of doing parsing, downloading, or anything that takes more than a fraction of a second.